consensus/src/validation/step.rs

```diff
@@ -176,6 +187,46 @@ impl<T: Operations + 'static, D: Database> ValidationStep<T, D> {
     }
 }
 
+pub fn validate_blobs(tx: &Transaction) -> Result<(), BlobError> {
+    if let Some(blobs) = tx.inner.blob() {
+        // TODO: Move this to vm::blob_check to perform the same checks
+        // while executing a tx directly from within a Block
+        match blobs.len() {
+            0 => Err(BlobError::BlobEmpty),
+            n if n > 6 => Err(BlobError::BlobTooMany(n)),
+            _n => {
+                // TODO: Add checks for gas price and gas limit
+                Ok(())
+            }
+        }?;
+        for blob in blobs {
```

Comment:
```suggestion

        for blob in blobs {
```

Reply:
I cannot find the right emoticon for this